File: clientpro.php
<?php

class Client
{
    public $name;
    public $account_number;

    function __construct($name, $account_number)
    {
        $this->name = $name;
        $this->account_number = $account_number;
    }
}

class ClientProfessionel extends Client
{
    public $siretNumber;

    function __construct($name, $account_number, $siretNumber)
    {
        parent::__construct($name, $account_number);
        $this->siretNumber = $siretNumber;
    }
}

$new_client = new Client("vanessa", "13489757659");
echo ($new_client->name . ' ' . $new_client->account_number);
$new_client_pro = new ClientProfessionel("gerard", "13477479575", "2746748575859");
echo (' ' . $new_client_pro->name . ' ' . $new_client_pro->account_number . ' ' . $new_client_pro->siretNumber);

File: clientpro.java
class Playground {
    public static void main(String[] args) {
        Client newClient = new Client("vanessa", "0984675859");
        System.out.println(newClient.name + ' ' + newClient.accountNumber);
        ClientPro newClientPro = new ClientPro("benoit", "09848595008", "09577585400948");
        System.out.println(newClientPro.name + ' ' + newClientPro.accountNumber + ' ' + newClientPro.siretNumber);
    }
}

class Client {

    public String name;
    public String accountNumber;

    Client(String name, String accountNumber) {
        this.name = name;
        this.accountNumber = accountNumber;
    }
}

class ClientPro extends Client {
    public String siretNumber;

    ClientPro(String name, String accountNumber, String siretNumber) {
        super(name, accountNumber);
        this.siretNumber = siretNumber;
    }
}